about.html

```diff
@@ -12,53 +12,87 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
+    <div class="container">
+    <nav class="topnav">
+       <img src="img/lambda-black.png" 
+                alt="Lambda icon" class="lambdaicon">
+                <a href="#">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+    </div> 
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+    <img src="img/jumbo-about.png" alt="skyline" class="jumboskyline">
 
+    <div class="containerabout-page">
+   
+   <h2>About</h2>
 
+    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
```

Comment:
Just a reminder to always have text wrapped in a text element `p`, `h1`, `h2`, `h3`, etc.